javaparser-core/src/main/java/com/github/javaparser/printer/SourcePrinter.java

```diff
@@ -78,14 +79,19 @@ public SourcePrinter println() {
         indented = false;
         return this;
     }
-    
+
     private StringBuilder bufAppend(final String arg) {
         updateCursor(arg);
         return buf.append(arg);
     }
 
+    private static final Pattern SPLIT_NEWLINE_PATTERN;
+    static {
+        SPLIT_NEWLINE_PATTERN = Pattern.compile("\r\n|\r|\n");
```

Comment:
Egh, end of line characters defined somewhere in the code. Will have to look into that later.

Reply:
I haven't found such string somewhere in the code base. Tell me if you find something :)